framework/test/mocha/functional/http/get/accounts/accounts.js

```diff
@@ -302,6 +303,9 @@ describe('GET /accounts', () => {
 
 			it('using no sort return accounts sorted by both balance and address in asending order as default behavior', async () => {
 				return accountsEndpoint.makeRequest({}, 200).then(res => {
+					_.each(res.body.data, item => {
```

Comment:
Don't you need to assign this? I don't see any reference in the Lodash documentation about `each` mutating the original array

Reply:
Yes it does!

Reply:
ohh ok and res.body.data is an object!

Reply:
Check next commit, implemented safer sorting without `_.each`